shell/platform/embedder/embedder.h

```diff
@@ -1660,6 +1660,30 @@ FLUTTER_EXPORT
 FlutterEngineResult FlutterEngineNotifyLowMemoryWarning(
     FLUTTER_API_SYMBOL(FlutterEngine) engine);
 
+//------------------------------------------------------------------------------
+/// @brief      Updates the timezone conversion routines used by the Flutter
+///             Engine.
+///
+/// @attention  On all platforms currently supported by Flutter, timezone
+///             information is read from /etc/timezone (and its various
+///             fall-backs). The engine has no mechanisms to watch for updates
+///             to these resources. If the embedder has access to such
+///             mechanisms, it must call this method to reinitialize process
+///             local locale specific data. Currently, the only argument to this
+///             method is ignored and may be nullptr. If, in the future, the
+///             engine decides to cache locale specific information per
+///             instance, it would need to invalidate these caches. For forward
+///             compatibility, it is recommended that a valid engine instance be
```

Comment:
A running valid engine, right?  Later we refer to it as a "running engine".

Reply:
Right. Since a non-running engine instance will not be returned from the Embedder API and there is no provision to pause an engine instance, I have been using valid and running interchangeably. I'll clear it up.